Cue/app/tabs/library/DeckView.js

```diff
@@ -92,6 +94,34 @@ class DeckView extends React.Component {
     this.props.flagCard(this.props.deckUuid, cardUuid, flag)
   }
 
+  _rateDeck = () => {
+    Alert.alert(
+      (Platform.OS === 'android'
+        ? 'Rate this deck'
+        : 'Rate This Deck'),
+      'Would you recommend this deck to others?',
+      [
+        {text: 'Cancel', style: 'destructive'},
+        {text: 'Would not Recommend', onPress: () => this.props.rateDeck(this.state.deck.uuid, -1)},
+        {text: 'Recommend',  onPress: () => this.props.rateDeck(this.state.deck.uuid, 1)},
+      ],
+
+    )
+  }
+
+  _copyDeck = () => {
+    Alert.alert(
+      (Platform.OS === 'android'
+        ? 'Are you sure you want to copy this deck?'
+        : 'Are You Sure You Want to Copy This Deck?'),
+      'Copying will create a private version of this deck in your library',
```

Comment:
Maybe add a note that this copy won't receive any updates made to the deck by the original owner.